src/Plugins/ComplexCore/src/ComplexCore/Filters/Algorithms/FindArrayStatistics.cpp

```diff
@@ -64,7 +64,7 @@ class FindArrayStatisticsByIndexImpl
     std::vector<uint64> length(numCurrentFeatures, 0);
     std::vector<T> min(numCurrentFeatures, std::numeric_limits<T>::max());
     std::vector<T> max(numCurrentFeatures, std::numeric_limits<T>::min());
-    std::vector<T> summation(numCurrentFeatures, 0);
+    std::vector<uint64> summation(numCurrentFeatures, 0);
```

Comment:
@joeykleingers Can I ask why you chose uint64? I think everywhere else that we are dealing with a summation we use float32 (and the summation data array itself is a float32 array) so would it be better to use a float32 here instead since we are casting it to float32 below anyway?

Reply:
I was following what the length data array does.  It is also float32 but the array used to calculate it is uint64.  We could probably change both.